Adds Analytics pages introduction section (#3746)
wso2/docs-apim: en/docs/observe/api-manager-analytics/analytics-pages/analytics-pages-introduction.md

# Analytics Pages Introduction

> New cloud analytics portal consists of several pages where pages are divided based on different stats(Ex: traffic

Reviewer: ```suggestion
The new cloud analytics portal consists of several pages where pages are divided based on different stats (e.g., traffic
```

> , latency). There are set of common controls which will govern the data shown in pages. Each page will refer these

Reviewer: ```suggestion
, latency). There are a set of common controls that enable you to govern the data shown in pages. Each page will refer these
```

 common controls and page specific controls to visualize the information requested by the user.
 
## Common Controls
### Time Picker
![APIM Analytics TIme Picker]({{base_path}}/assets/img/observe/time-picker.png)

Time picker is located in the top left hand corner of the analytics dashboard. Once you choose a given time frame

> , all dashboards will use that time range when you move between different pages. However if you reload the page then

Reviewer: ```suggestion
, all dashboards will use that time range when you move between different pages. However, if you reload the page then
```

>  time picker will be reverted to the default setting which is 'Past 24 Hours'.

Reviewer: ```suggestion
 the time picker will revert to the default setting, which is 'Past 24 Hours'.
```

>  Time picker consist of several pre-defined time intervals. Below image illustrates those predefined time intervals.

Reviewer: ```suggestion
 The time picker consists of several pre-defined time intervals. The following image illustrates those predefined time intervals.
```

 
 ![APIM Analytics TIme Picker List]({{base_path}}/assets/img/observe/time-picker-list.png)
 

>  You can select any of these pre-defined intervals to view stats relevant to that time frame. However granularity of

Reviewer: ```suggestion
 You can select any of these pre-defined intervals to view stats relevant to that time frame. However, granularity of
```

>   each time range is pre-defined too and you can't change the granularity of a pre-defined range. 

Reviewer: ```suggestion
  each time range is pre-defined too and you cannot change the granularity of a pre-defined range. 
```

>   If you need to select a custom time range with a custom granuarity, Then you have to use the custom selector which

Reviewer: ```suggestion
  If you need to select a custom time range with a custom granularity, you have to use the custom selector that
```

>    is show below.

Reviewer: ```suggestion
   is shown below.
```

   
![APIM Analytics TIme Picker List]({{base_path}}/assets/img/observe/time-picker-custom.png)

In the custom picker first you have to select the granularity that you need to view the data. Then from the calendar

>  widget you can select the date range and if applicable clock widget will be also enabled to select the time. However,

Reviewer: ```suggestion
 widget you can select the date range and if applicable the n clock widget will be also enabled to select the time. However,
```

  there are certain restrictions applied on the range that you can view based on the selected granularity
  . Restrictions are based on the length of the selected range and time from current date to starting date of
   selected range. Time picker will automatically apply these restrictions. If you need to look at the restrictions
    in details, please visit deep dive section of our docs.
    
### Environment Picker

![APIM Analytics Environment Picker List]({{base_path}}/assets/img/observe/environment-picker.png)

This widget allows you to select the environment you need to view statistics for. When you are 
[Registering for Analytics]({{base_path}}/observe/api-manager-analytics/configure-analytics/register-for-analytics) 

> at the On-Prem Key generation step, you are required to provide an 'Key Name'. This serves as your environment name

Reviewer: ```suggestion
at the On-Prem Key generation step, you are required to provide a 'Key Name'. This serves as your environment name
```

. Suppose you have two different environments for two different business units of your organization. In such a case
 you can generate two On-Premise Keys and add them to respective Gateways of each deployment. Once that is done
  'Environment Picker' will show two options and based on the selection dashboards will only show statistics relevant

>    to selected environment.

Reviewer: ```suggestion
   to the selected environment.
```

   
### Tenant Picker

![APIM Analytics Tenant Picker List]({{base_path}}/assets/img/observe/tenant-picker.png)

Tenant Picker will be enabled if your selected environment uses multi-tenancy. If multi-tenancy is enabled then using

>  this picker you have to select the tenant that you need to observe abalytics data. When tenancy is enabled

Reviewer: ```suggestion
 this picker you have to select the tenant that you need to observe analytics data. When tenancy is enabled
```

>  , data will only be show in per tenant basis. If you want to view cumulative analytics data for all tenants, Then

Reviewer: ```suggestion
 , data will only be shown per tenant basis. If you want to view cumulative analytics data for all tenants,
```

>   you have to select **carbon.super** as the tenant which is the default super tenant name. All tenant data will be

Reviewer: ```suggestion
  you have to select **carbon.super** as the tenant, which is the default super tenant name. All tenant data will be
```

>    aggregated and show under super tenant.

Reviewer: ```suggestion
   aggregated and shown under super tenant.
```

   
## Pages

### Overview Page
This is the welcome page of the analytics portal. This page will mainly display a quick overview of the whole
 deployment. This page is targeted towards providing business users with a quick overview. For more information
  please refer [this]({{base_path}}/observe/api-manager-analytics/analytics-pages/analytics-pages-overview) 
  documentation.
  
### Traffic Page
This page displays all traffic related information available in the system such as API Usage, Application Usage

> , Resource Usage and etc. For more information please refer 

Reviewer: ```suggestion
, Resource Usage, etc. For more information please refer 
```

[this]({{base_path}}/observe/api-manager-analytics/analytics-pages/analytics-pages-traffic) documentation.

### Errors Page

> Error page displays analytics stats related to erroneous API calls happened within the system. It will include error

Reviewer: ```suggestion
The error page displays analytics stats related to erroneous API calls that happened within the system. It will include error
```

>  categorization based on the error type and then further drill down into sub types. This page can be used as the

Reviewer: ```suggestion
 categorization based on the error type and then further drills down into sub types. This page can be used as the
```

  starting point for debugging API errors. For more information please refer 
  [this]({{base_path}}/observe/api-manager-analytics/analytics-pages/analytics-pages-erros) documentation.
  
### Latency Page 
This page illustrates API latency related information such as slowest APIs, latency by category, etc. This page is a
 good starting point if you are investigating about slowness of the system of any particular API. For more
  information please refer 
  [this]({{base_path}}/observe/api-manager-analytics/analytics-pages/analytics-pages-latency) documentation.
  
### Cache Page

> This page visualizes details API caching functionality including cache hit rate, hit percentage, etc. You can use

Reviewer: ```suggestion
This page visualizes details on API caching functionality including cache hit rate, hit percentage, etc. You can use
```

>  this page to decide the effectiveness of API caching and also as additional information then investigating API

Reviewer: ```suggestion
 this page to decide the effectiveness of API caching and also as additional information when investigating API
```

  slowness. For more information please refer 
  [this]({{base_path}}/observe/api-manager-analytics/analytics-pages/analytics-pages-cache) documentation.
  
### Devices Page

This page displays information about OS platforms and HTTP Agents end users are using to invoke the APIs. You can use

>  this page to get and idea of the distribution of your user-base and improve your APIs to match the audience. For

Reviewer: ```suggestion
 this page to get an idea of the distribution of your user-base and improve your APIs to match the audience. For
```

  more information please refer 
  [this]({{base_path}}/observe/api-manager-analytics/analytics-pages/analytics-pages-devices) documentation.
  
### Alerts Page

> This page illustrated information related to alerts issued by the system. You can use this page as health monitoring

Reviewer: ```suggestion
This page illustrates information related to alerts issued by the system. You can use this page as a health monitoring
```

>  dashboard and make visible to whole team. Using the page you can get more information about issues alerts. This page

Reviewer: ```suggestion
 dashboard and make it visible to whole team. Using the page you can get more information about issues alerts. This page
```
